src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -70,22 +70,28 @@
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({CheckstyleAntTask.class, Closeables.class})
-public class CheckstyleAntTaskTest extends BaseCheckTestSupport {
+public class CheckstyleAntTaskTest extends AbstractPathTestSupport {
 
     private static final String FLAWLESS_INPUT =
-            "/ant/checkstyleanttask/InputCheckstyleAntTaskFlawless.java";
+            "InputCheckstyleAntTaskFlawless.java";
     private static final String VIOLATED_INPUT =
-        "ant/checkstyleanttask/InputCheckstyleAntTaskError.java";
+            "InputCheckstyleAntTaskError.java";
     private static final String WARNING_INPUT =
-        "ant/checkstyleanttask/InputCheckstyleAntTaskWarning.java";
+            "InputCheckstyleAntTaskWarning.java";
     private static final String CONFIG_FILE =
-            "ant/checkstyleanttask/InputCheckstyleAntTaskTestChecks.xml";
-    private static final String CONFIG_RESOURCE = "/com/puppycrawl/tools/checkstyle/" + CONFIG_FILE;
+            "InputCheckstyleAntTaskTestChecks.xml";
+    private static final String CONFIG_RESOURCE =
+            "/com/puppycrawl/tools/checkstyle/ant/checkstyleanttask/" + CONFIG_FILE;
```

Comment:
this variable is used only ones, please inline it to code and reuse `getPackageLocation` to avoid copy-paste of path